_docs_integrate/error-codes.md

```diff
@@ -24,74 +24,163 @@ required_by:
+| {% include anchor a="error.runtime.notStarted" %}                                                                   | The runtime is not started. You can only stop the runtime if you executed start before.                                                                                                                                                                                                       |
+| {% include anchor a="error.runtime.notSupported" %}                                                                 | The requested feature is not supported.                                                                                                                                                                                                                                                       |
+| {% include anchor a="error.runtime.recordNotFound" %}                                                               | The requested Record was not found. Make sure the ID exists and the record is not expired.                                                                                                                                                                                                    |
+| {% include anchor a="error.runtime.recordNotFoundWithMessage" %}                                                    | The requested Record was not found. Make sure the ID exists and the record is not expired.                                                                                                                                                                                                    |
+| {% include anchor a="error.runtime.relationshipTemplates.cannotCreateTokenForPeerTemplate" %}                       | You cannot create a token for a RelationshipTemplate of a peer. ([Token]({% link _docs_integrate/data-model-overview.md %}#token), [Create Token for own RelationshipTemplate]({% link _docs_use-cases/use-case-transport-create-token-for-own-relationshiptemplate.md %}))                   |
+| {% include anchor a="error.runtime.relationshipTemplates.cannotCreateQRCodeForPeerTemplate" %}                      | You cannot create a QR code for a RelationshipTemplate of a peer. ([Get RelationshipTemplate]({% link _docs_use-cases/use-case-transport-get-relationshiptemplate.md %}))                                                                                                                     |
+| {% include anchor a="error.runtime.relationshipTemplates.invalidReference" %}                                       | The given reference is not valid. The reference for a RelationshipTemplate must start with 'VE9L' or 'VE9L'.                                                                                                                                                                                  |
```

Comment:
Again interesting strings. But this time they are equal. Would be create if you could investigate further.

Reply:
Here you are right, it must start with 'UkxU'.